### DISCUSSION TITLE
Tina not commiting media to repository

### DISCUSSION THREAD

**Discussion by KosisoPlato** [tinacms/tinacms#3966] [Help]
Please I'm working on a Gatsby project and I've set the media publicFolder to "static" and left the mediaRoot as an empty string "", but Tina is still not commiting any of the media into my project repository for any of my uploads.

**Comment by kldavis4** [comment]
@KosisoPlato can you jump on our discord https://discord.gg/ztNCGE8b and post this question there in the #help channel? It will be easier to assist you there.

**Reply by KosisoPlato** [reply]
@kldavis4 I've tried joining the discord but I keep getting this message
<img width="508" alt="Screenshot 2023-05-24 at 19 50 33" src="https://github.com/tinacms/tinacms/assets/133699026/89abd6e4-cbe6-49da-ab51-12c119a3fc4b">

**Reply by kldavis4** [reply]
Try this https://discord.com/invite/zumN63Ybpf
